### PR TITLE
Add deprecated warning for RandomResizedCrop

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

File: mmaction/datasets/pipelines/augmentations.py [hunk 1]
```diff
@@ -639,6 +640,9 @@ def __init__(self,
                  area_range=(0.08, 1.0),
                  aspect_ratio_range=(3 / 4, 4 / 3),
                  lazy=False):
+        warnings.warn('"RandomResizedCrop" is deprecated since it is provided '
```
Comment (innerlee) [inline mmaction/datasets/pipelines/augmentations.py:643]: "RandomResizedCrop" is deprecated and will be removed in later versions. It is currently not used in MMAction2.

Comment (kennymckormick) [reply mmaction/datasets/pipelines/augmentations.py:643]: Emmm, actually I mean RandomScale is deperacated. Please check my last PR.

Comment (kennymckormick) [reply mmaction/datasets/pipelines/augmentations.py:643]: RandomResizedCrop is OK.

Comment (dreamerlin) [reply mmaction/datasets/pipelines/augmentations.py:643]: = =, my fault

**Pull request closed without merging**